lib/Controller/PageController.php

```diff
@@ -39,7 +39,7 @@ public function index() {
         $templateResponse = new TemplateResponse(
             $this->appName,
             'index',
-            $this->fileModel->getFileMimeTypesInfo()
+            $this->fileModel->getFileSortingAsSize()

```

Comment:
I guess, we should merge these 2 functions in another function before return to model.

Reply:
I managed to merge those two functions in another function that returns a multidimensional array which includes return values of those two functions